src/Calculator.js

```diff
@@ -0,0 +1,65 @@
+import { MissionUtils } from "@woowacourse/mission-utils";
+
+class Calculator {
+  async start() {
+    const input = await this.getInput();
+    try {
+      const result = this.calculate(this.parseInt(input));
+      this.printResult(result);
+    } catch (error) {
+      throw error;
+    }
+  }
+
+  async getInput() {
+    const input = await MissionUtils.Console.readLineAsync(
+      "덧셈할 문자열을 입력해 주세요.\n"
+    );
+    return input;
+  }
```

Comment:
```suggestion
  getInput() {
    return MissionUtils.Console.readLineAsync("덧셈할 문자열을 입력해 주세요.\n");
  }
```
`getInput` 로직을 이렇게 축약하고 `async/await` 키워드를 제거하는건 어떻게 생각하시나요??

Reply:
한번에 return하는게 더 보기 깔끔하고 축약되어 좋은 것 같습니다! 
앞으로는 이렇게 작성해볼게요